todo_project/utils/health.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from todo_project.db.config import get_database_client
+from pymongo.errors import ConnectionFailure
+
+logger = logging.getLogger(__name__)
+
+
+def is_db_healthy():
```

Comment:
rename  this function name something like check_database_status

Reply:
Renames this to `check_database_health`